Gems/Atom/Feature/Common/Assets/Shaders/Reflections/ReflectionScreenSpaceRayTracingClosestHitProcedural.azsl

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "ReflectionScreenSpaceRayTracingClosestHit.azsli"
+#include <Atom/Features/RayTracing/RayTracingIntersectionAttributes.azsli>
+
+// TODO(intersection): Implement multiple closesthit-shaders per source file
```

Comment:
This would require some code changes in Vulkan-RayTracingPipelineState (in DX12 it works fine). In the spirit of keeping this PR small I did not add this to this PR.